### PR TITLE
Update Simplified Chinese Translations

### PR DESCRIPTION
**Pull request by wzsx150**
I have updated all the Simplified Chinese translations on Crowdin, making numerous revisions.
I've committed the updated content to this repository and hope the commit can be merged.

This commit only updates the Simplified Chinese translations and does not affect the stability of Ryujinx.
I have carefully translated it in accordance with the original English text, so there shouldn't be any issues.
I highly recommend that it be directly merged into the mainline.

Thank you very much!

### CODE REVIEW

**Comment by gdkchan** [comment]
We sync the Crowdin translations with the GitHub repo every once in a while, so you don't need to create a PR to do that yourself. FWIW there is another PR that touches those translations: #6388.

**Comment by gdkchan** [comment]
Superseded by #6541.

**Comment by wzsx150** [comment]
> We sync the Crowdin translations with the GitHub repo every once in a while, so you don't need to create a PR to do that yourself. FWIW there is another PR that touches those translations: #6388.

Sorry, I've only recently started paying attention to Ryujinx and noticed that there are many omissions and errors in the Simplified Chinese translation. Therefore, I wanted to contribute my part, so I updated the translations on Crowdin.

I've noticed that the synchronization between Crowdin and GitHub doesn't seem very timely. To correct the translation issues more quickly, that's why I created this PR.

I've also seen the PR #6388, which has only a few modifications. Regarding his modifications to certain translations, we just have slightly different understandings, which won't affect user experience. I hope he can decide whether to continue modifying the Simplified Chinese translation after rebasing.

In summary, I really like Ryujinx and hope it will get better and better. Also, I'm very grateful to you for timely updating the translations. You're truly an excellent and efficient team.

**Comment by gdkchan** [comment]
I changed the Crowdin configuration today, so it should sync translations every day now (as long something was changed).

**Comment by wzsx150** [comment]
> I changed the Crowdin configuration today, so it should sync translations every day now (as long something was changed).

I have already updated the Simplified Chinese translation on Crowdin, but it seems that the changes have not automatically updated to this repository.

**Comment by gdkchan** [comment]
Its on #6550, it was not merged yet.
